source/component/PasDoc_Items.pas

```diff
@@ -2701,6 +2803,7 @@ procedure TPasUnits.SetUnitAt(const AIndex: Integer; const Value: TPasUnit);
 constructor TPasRoutine.Create;
 begin
   inherited;
+  FParamTypes := TStringVector.Create;
 end;
```

Comment:
From a quick glance, this may be a memory leak -- you never do `FreeAndNil(FParamTypes)` .